packages/PreviewCard/PreviewCard.jsx

```diff
@@ -0,0 +1,160 @@
+
+import warn from '../../shared/utils/warn'
+
+const Anchor = styled.a`
+  text-decoration: none;
+  cursor: pointer;
+`
+
+const BoxContainer = styled.div`
+  background-color: white;
+  width: 100%;
+  opacity: 1;
+  border: 1px solid ${colorWhiteLilac};
+  border-radius: 4px;
+  height: ${props => (props.header || props.footer ? '465px' : '350px')};
+  @media (max-width: 576px) {
+    height: auto;
+  }
+  overflow: hidden;
+  &:hover {
+    box-shadow: 0 0 16px 0 rgba(213, 213, 213, 0.5);
+  }
+  display: inline-block;
+`
+
+const ContentContainer = styled.div`
+  line-height: ${props => (props.header && props.footer ? '21px' : undefined)};
+`
+
+const ImageContainer = styled.div`
+  border-radius: 4px 4px 0 0;
+  overflow: hidden;
+  line-height: 0px;
+  max-height: 250px;
+  @media (max-width: 992px) {
+    height: auto;
+  }
+  img {
+    transition: all 0.3s ease-in-out !important;
+  }
+  &:hover img {
+    transform: scale(1.03);
+  }
+`
+
+const P = styled.p`
+  color: ${colorTelusPurple};
+`
+
+/**
+ * The PreviewCard component creates the appearance of a page snippet, and can be used in a list format.
+ * @version ./package.json
+ */
+const PreviewCard = ({ category, other, image, body, footer, href, linkComponent, ...rest }) => {
+  if (rest.to && !(linkComponent && rest.to)) {
+    warn('Link', 'The props `linkComponent` and `to` must be used together.')
+  }
+
+  let newBody = body
+  if (body.length > 70) {
+    newBody = `${body.substr(0, 70)}...`
+  }
+
+  let header = category || other
+  if (category && other) {
+    header = `${category} \u00B7 ${other}`
+  }
+
+  const innerCard = (
+    <BoxContainer header={header} footer={footer}>
+      {image && (
+        <ImageContainer header={header} footer={footer}>
+          {image}
+        </ImageContainer>
+      )}
+      <Box horizontal={header || footer ? 4 : 3} vertical={header || footer ? 5 : 3}>
+        <ContentContainer header={header} footer={footer} data-testid="contentContainer">
+          {header && (
+            <Box>
+              <Text size="small" bold>
+                {header}
+              </Text>
+            </Box>
+          )}
+          <Box vertical={header || footer ? 3 : undefined}>
+            <P alt={body}>{newBody}</P>
+          </Box>
+          {footer && (
+            <Box>
+              <Text size="small">{footer}</Text>
+            </Box>
+          )}
+        </ContentContainer>
+      </Box>
+    </BoxContainer>
+  )
+
+  return React.createElement(
+    linkComponent || Anchor,
+    {
+      ...safeRest(rest),
+      href,
+    },
+    innerCard
+  )
+}
+
+PreviewCard.propTypes = {
+  /**
+   * Image component that will appear at the top of the card, above the content section.  Recommended dimensions is 369x269px.
+   */
+  image: PropTypes.node.isRequired,
+  /**
+   * Text that will appear at the top of the content section.  Recommended to be only one or two words.
+   */
+  category: PropTypes.string,
```

Comment:
it's not very clear that _category_ and _other_ map to title. Would it rather be better to combine the two as _title_ instead and let the user pass the strings combined. So currently
```
<PreviewCard category="Data Intelligence" other="March 21, 2019" />
```
It would be much clearer if it were
```
<PreviewCard title="Data Intelligence \u00B7 March 21, 2019"
```
What benefit do you get when you split _title_ into _category_ and _other_? I would argue that it make it harder for the user who consumes the app since it's not obvious that _category_ and _other_ map to _title_. Also It's not really hard for the client to set the title as they see fit. 

Maybe a compromise would be _titleCategory_ and _titleOther_, that way it would be easier for client to know these props belong in the _title_. However I would prefer just _title_. Really simplifies the code as well.

Reply:
After speaking with the designer, we do feel it would be better for more flexibility with the title, and the only benefit with separating `category` and `other` would be appending the dot.  But since the user consuming the `PreviewCard` can append the dot him/herself, then we will combine the two categories.

Reply:
I also named it `header` to be consistent with the other props: `body` and `footer`